packages/pages/worker/index.js

```diff
@@ -0,0 +1,179 @@
+const parser = require('ua-parser-js');
+const semver = require('semver');
+
+addEventListener('fetch', event => {
+	event.respondWith(handleRequest(event.request))
+})
+
+/**
+ * Respond to the request
+ * @param {Request} request
+ */
+async function handleRequest(request) {
+	const resp = await fetch('https://mrhenry.github.io/core-web' + new URL(request.url).pathname);
+
+	if (!request.headers.get('User-Agent')) {
+		return cleanupIfNoMatch(resp);
+	}
+
+	const ua = parser(request.headers.get('User-Agent'));
+	if (!ua || !ua.browser) {
+		return cleanupIfNoMatch(resp);
+	}
+
+	let uaBrowserName = ua.browser.name.toLowerCase();
+	if (uaBrowserName === 'mobile safari') {
+		uaBrowserName = 'safari';
+	}
+
+	let possibleTargets = targets.filter((target) => {
+		if (target.browsers[uaBrowserName] && semver.satisfies(semver.coerce(ua.browser.major), '>= ' + target.browsers[uaBrowserName])) {
+			return true;
+		}
+
+		return false;
+	});
+
+	if (!possibleTargets.length) {
+		return cleanupIfNoMatch(resp);
+	}
```

Comment:
```suggestion
```

Legacy should probably be used as the fallback. Most of the time these user-agents will either support something like a legacy/fallback target (ex. Pale Moon) or they will not execute JS at all.